examples/bigcommerce-catalyst/app/(default)/[...path]/page.tsx

```diff
@@ -1,28 +1,28 @@
-import { getSiteVersion } from '@makeswift/runtime/next/server'
-import { notFound } from 'next/navigation'
-import { Page as MakeswiftPage } from '@makeswift/runtime/next'
+import { getSiteVersion } from '@makeswift/runtime/next/server';
+import { notFound } from 'next/navigation';
+import { Page as MakeswiftPage } from '@makeswift/runtime/next';
 
-import { client } from '~/lib/makeswift/client'
-import '~/lib/makeswift/components'
+import { client } from '~/lib/makeswift/client';
+import '~/lib/makeswift/components';
 
-type ParsedUrlQuery = { path?: string[] }
+type ParsedUrlQuery = { path?: string[] };
 
 export async function generateStaticParams() {
-  const pages = await client.getPages()
-
-  return pages.flatMap((page) => ({
+  return await client
+    .getPages()
+    .map((page) => ({
       path: page.path.split('/').filter((segment) => segment !== ''),
-    }),
-  )
+    }))
+    .toArray();
 }
 
 export default async function Page({ params }: { params: ParsedUrlQuery }) {
-  const path = '/' + (params?.path ?? []).join('/')
+  const path = '/' + (params?.path ?? []).join('/');
   const snapshot = await client.getPageSnapshot(path, {
     siteVersion: getSiteVersion(),
-  })
+  });
```

Comment:
Semicolons ^^

Reply:
The catalyst example has a separate prettier config, so the auto-formatting is part of the change - the rest of the files in this example have semicolons. I believe this file somehow made it past auto-formatting.